cnxpublishing/views/admin.py

```diff
@@ -516,7 +517,11 @@ def get_baking_statuses_sql(get_request):
                        f.sha1 as recipe,
                        m.module_ident,
                        ident_hash(m.uuid, m.major_version, m.minor_version),
-                       bpsa.created, ctm.status as state, ctm.traceback
+                       bpsa.created, ctm.traceback,
+                       CASE WHEN ctm.status = 'SUCCESS' and m.stateid = 8
```

Comment:
add a `-- state = fallback` to the end of this line just to make it clear what `stateid = 8` means.

Reply:
Or as you pointed out in chat... join modulestates and match on the name.